applications/fax/src/fax_worker.erl

```diff
@@ -288,52 +287,45 @@ handle_cast('prepare_job', #state{job_id=JobId
                                  ,job=JObj
                                  }=State) ->
     send_status(State, <<"fetching document to send">>, ?FAX_PREPARE, 'undefined'),
-    case fetch_document(JObj) of
-        {'ok', 200, RespHeaders, RespContent} ->
+    case fetch_document(JobId, JObj) of
+        {'ok', Filepath, {PageCount, Size}} ->
             send_status(State, <<"preparing document to send">>, ?FAX_PREPARE, 'undefined'),
-            case prepare_contents(JobId, RespHeaders, RespContent) of
-                {'error', Cause} ->
-                    send_error_status(State, Cause),
-                    {Resp, Doc} = release_failed_job('bad_file', Cause, JObj),
+            Values = [{<<"pvt_pages">>, PageCount}
+                     ,{<<"pvt_size">>, Size}
+                     ],
+            case PageCount of
+                Num when Num == 0 ->
+                    _ = send_error_status(State, 'bad_file'),
```

Comment:
`send_error_status/2` asks for a `ne_binary()` as the status; why use an `atom()` here?